src/posts/2022-11-04-kubecon-wrapup/index.md

```diff
@@ -0,0 +1,60 @@
+---
+path: '/blog/2022/11/04/cilium-kubecon-na-wrapup'
+date: '2022-11-04T17:00:00.000Z'
+title: 'Cilium at KubeCon NA 2022 Wrap Up'
+isPopular: false
+isFeatured: false
+ogImage: kubecon_stage.webp
+ogSummary: "Find out the buzz from around the community at KubeCon"
+categories:
+  - Community
+tags:
+  - Cilium
+  - KubeCon
+---
+
+*November 4th, 2022*  
+*Author: Bill Mulligan, Isovalent*
+
+KubeCon + CloudNativeCon NA 2022 was a buzz with activity around Cilium. In this post, I will try to sum up all the big annoucements and moments so the people that weren't able to attend in person can feel like they were there too.
+
+
+### Announcement Microsoft AKS picking Cilium
+
+Cilium is now the default CNI for all major cloud providers. If it wasn't obvious already, Cilium has become the standard networking layer for Kubernetes everywhere. You can read the blog from Isovalent about it [here](https://isovalent.com/blog/post/azure-cni-cilium/). Microsoft has an [announcement](https://azure.microsoft.com/en-us/updates/public-preview-azure-cni-powered-by-cilium/), [blog](https://techcommunity.microsoft.com/t5/azure-networking-blog/azure-cni-powered-by-cilium-for-azure-kubernetes-service-aks/ba-p/3662341), and [docs](https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium) too!
```

Comment:
```suggestion
Cilium is now the default CNI for all major cloud providers. If it wasn't obvious already, Cilium has become the standard networking layer for Kubernetes everywhere. Here's the [announcement](https://azure.microsoft.com/en-us/updates/public-preview-azure-cni-powered-by-cilium/) from Microsoft, [blog](https://techcommunity.microsoft.com/t5/azure-networking-blog/azure-cni-powered-by-cilium-for-azure-kubernetes-service-aks/ba-p/3662341), and [docs](https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium). There's also a blog from Isovalent about it [here](https://isovalent.com/blog/post/azure-cni-cilium/). 
```

I think this should emphasise the MSFT links first, and then the commentary on Isovalent's blog